server/player/handler.go

```diff
@@ -22,6 +22,8 @@ type Handler interface {
 	HandleMove(ctx *event.Context, newPos mgl64.Vec3, newYaw, newPitch float64)
 	// HandleTeleport handles the teleportation of a player. ctx.Cancel() may be called to cancel it.
 	HandleTeleport(ctx *event.Context, pos mgl64.Vec3)
+	// HandleToggleSneak handles when the player starts or stops sneaking
+	HandleToggleSneak(ctx *event.Context, toggle bool)
```

Comment:
Same with this 'toggle' field as in the HandleToggleSprint PR. This is quite an ambiguous variable name.

Reply:
I think something like `toggled` would be better.

Reply:
I think that's still ambiguous. Does it mean the old or the new value?

Reply:
The new value. Is something like `isToggled` or `currentlyToggled` better? I'm not entirely sure what would be good to tell it's a new value without making it paragraphs long.

Reply:
See the toggle sprint PR, I think that's the most straightforward way to do it.

Reply:
Or maybe a `sneakingAfter` field.

Reply:
`old, new` is better than `before, after` I think.